flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/FineGrainedSlotManager.java

```diff
@@ -64,6 +64,9 @@
 public class FineGrainedSlotManager implements SlotManager {
     private static final Logger LOG = LoggerFactory.getLogger(FineGrainedSlotManager.class);
 
+    // We currently make the delay of requirements check a constant time. This delay might be
+    // configurable by user in the future.
+    private static final long REQUIREMENTS_CHECK_DELAY_MS = 50L;
```

Comment:
I'm ok with not exposing this to user as the first step. However, I would suggest to move the constant to `ResourceManagerRuntimeServices`, so that we can pass different values for tests.